src/Makefile

```diff
@@ -320,15 +320,44 @@ ifeq ($(BUILD_TLS),module)
 	TLS_MODULE_CFLAGS+=-DUSE_OPENSSL=$(BUILD_MODULE) $(OPENSSL_CFLAGS) -DBUILD_TLS_MODULE=$(BUILD_MODULE)
 endif
 
+define MAYBE_UNINSTALL_REDIS_SYMLINK
+    @if [ $(USE_REDIS_SYMLINKS) = "yes" ]; then \
+		rm -f $(1)/$(subst $(ENGINE_NAME),redis,$(2)); \
+    fi
+endef
```

Comment:
If this is used more than once, then it can be even better is to define `MAYBE_UNINSTALL_REDIS_SYMLINK=UNINSTALL_REDIS_SYMLINK` in the first case and to define UNINSTALL_REDIS_SYMLINK separately (somewhere else).

Reply:
Very nice. Thanks for your help.